app/models/async_transaction/vet360/base.rb

```diff
@@ -2,6 +2,32 @@
 
 module AsyncTransaction
   module Vet360
-    class Base < AsyncTransaction::Base; end
+    class Base < AsyncTransaction::Base;
+
+      def self.refresh_transaction_status(user, tx_id = nil)
+
```

Comment:
Add a guard to not send a request to Vet360 if OUR `status` (not transaction status) is in a final state.